hmda/src/test/scala/hmda/api/http/admin/InstitutionAdminHttpApiSpec.scala

```diff
@@ -131,6 +180,25 @@ class InstitutionAdminHttpApiSpec extends AkkaCassandraPersistenceSpec with Must
       }
     }
 
+    "Create institution when it doesn't exist for the lei" in {
+      val testSampleInstitution =
```

Comment:
Same as my above comment, I think you can just copy teh sampleInstitution and use copy() to make a change to one field so it is a new institution.

Reply:
Agreed, fixed!